desci-server/src/utils/driveUtils.ts

```diff
@@ -393,6 +393,26 @@ export function DANGEROUSLY_addComponentsToManifest(
   return manifest;
 }
 
+export function prepareFirstNestingComponents(firstNestingComponents: FirstNestingComponent[]) {
+  const preparedComponents: ResearchObjectV1Component[] = [];
+  firstNestingComponents.forEach((c) => {
+    const comp = {
+      id: randomUUID(),
+      name: c.name,
+      ...(c.componentType && { type: c.componentType }),
+      ...(c.componentSubtype && { subtype: c.componentSubtype }),
```

Comment:
I find this syntax quite confusing… Why not simply `type: c.componentType`, if the check is for `undefined`, or `type: c.componentType || undefined` if it's checking for some other falsy value.

Is there some reason why the key absolutely cannot exist? If so, documenting this would be useful :)